Services/UI/OJS.Services.Ui.Business/Implementations/SubmissionsBusinessService.cs

```diff
@@ -456,14 +457,15 @@ public async Task Submit(SubmitSubmissionServiceModel model)
         await this.submissionsData.Add(newSubmission);
         await this.submissionsData.SaveChanges();
 
-        await this.submissionsForProcessingData.Add(newSubmission.Id);
+        submissionServiceModel = this.BuildSubmissionForProcessing(newSubmission, problem, submissionType);
```

Comment:
Can this be outside the transaction?

Reply:
When I put it outside of the transaction I get the same error in the database that with the Checkers table that we previously discussed.